### PR TITLE
Update call to action box text styling

### PR DESCRIPTION
**Pull request by Zhao-Andy**
## What type of PR is this? (check all applicable)
- [x] Bug Fix

## Description
Found another small styling issue 🙃 

Also added a comment and refactored a bit.

Screenshot of Before and After changes

## [optional] What gif best describes this PR or how it makes you feel?

Homer Simpson looks through a magnifying glass and says, "Aha!"

### CODE REVIEW

File: app/models/organization_membership.rb [hunk 1]
```diff
@@ -2,9 +2,11 @@ class OrganizationMembership < ApplicationRecord
   belongs_to :user
   belongs_to :organization
 
+  USER_TYPES = %w[admin member guest].freeze
```
Comment (mstruve) [inline app/models/organization_membership.rb]: 🔥

File: app/views/users/_organization.html.erb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <% if @organization_membership %>
   <%= render "users/org_#{@organization_membership.type_of_user}" %>
+  <%# "org_admin", "org_guest", "org_member", etc. For all types, see app/models/organization_membership.rb  %>
```
Comment (vaidehijoshi) [inline app/views/users/_organization.html.erb]: small nitpick: can we put the comment above the line that it refers to? that seems to be what we do elsewhere. for a second i didn't even realize that this was referencing the line above and thought it was just accidentally-commented-out code 🙈

Comment (Zhao-Andy) [reply app/views/users/_organization.html.erb]: Aha yep! I _knew_ shouldn't have broken convention here 😅

**Comment by Zhao-Andy** [comment]
Merged master since CI keeps failing 🤔 Specifically:
Screen Shot 2020-03-23 at 6 08 33 PM

Passes locally though. Sorry about the auto-dismiss!

**Pull request merged**